src/Quartz.Extensions.Hosting/IQuartzHostedServiceListener.cs

```diff
@@ -0,0 +1,8 @@
+namespace Quartz
+{
+    public interface IQuartzHostedServiceListener
```

Comment:
I think this interface might get troublesome as the error count is kind of ambiguous, but I don't have better ideas at the moment from the top of my head...